MantidQt/CustomInterfaces/test/EnggDiffFittingPresenterTest.h

```diff
@@ -361,7 +366,7 @@ class EnggDiffFittingPresenterTest : public CxxTest::TestSuite {
 
     // No errors/1 warnings. File entered is not found
     EXPECT_CALL(mockView, userError(testing::_, testing::_)).Times(0);
-    EXPECT_CALL(mockView, userWarning(testing::_, testing::_)).Times(1);
+    EXPECT_CALL(mockView, userWarning(testing::_, testing::_)).Times(2);
 
     pres.notify(IEnggDiffFittingPresenter::FittingRunNo);
```

Comment:
It would be good to have something like:
```
TSM_ASSERT( 
        "Mock not used as expected. Some EXPECT_CALL conditions were not satisfied",
        testing::Mock::VerifyAndClearExpectations(&mockView))
```
At the end of these tests too. The key here is to have an assert on `VerifyAndClearExpectations()`, otherwise the tests don't fail even if you see the warnings from gmock.
Other tests have it and I think that all / almost all should have it.